### PR TITLE
Adjusting the permissions node for the /rtp command

### PR DESCRIPTION
**Pull request by alazeprt**
Remove the original permission node huskhomes.command.rtp.world and add the permission node huskhomes.command.rtp.<world>, where <world> can be changed to any of the server's worlds, and this permission node can grant the player the permission to be randomly teleported to a specified world

### CODE REVIEW

**Comment by alazeprt** [comment]
Corresponds to issue [#625](https://github.com/WiIIiam278/HuskHomes/issues/625)

File: common/src/main/java/net/william278/huskhomes/command/RtpCommand.java [hunk 1]
```diff
@@ -29,18 +29,16 @@
...
 
     protected RtpCommand(@NotNull HuskHomes plugin) {
         super("rtp", List.of(), "[player] [world]", plugin);
-        addAdditionalPermissions(Map.of(
-                "other", true,
-                "world", true
-        ));
+        Map<String, Boolean> map = new HashMap<>();
+        map.put("other", true);
+        plugin.getWorlds().forEach(world -> map.put(world.getName(), true));
```
Comment (WiIIiam278) [inline common/src/main/java/net/william278/huskhomes/command/RtpCommand.java:40]: Can we filter this by `plugin.getSettings().getRtp().isWorldRtpRestricted(world)` (so as to not register unneccessary perms)?

**Review by WiIIiam278 — CHANGES_REQUESTED** [review]
This looks great, just need the above comment to be addressed :)

**Comment by alazeprt** [comment]
Resolved in the latest commit

**Review by WiIIiam278 — APPROVED** [review]
Cool, looks great then.

**Pull request merged**